config.json

```diff
@@ -1,6 +1,7 @@
 {
   "language": "Delphi Pascal",
   "active": true,
+  "blurb": "Delphi is a programming language and SDK for desktop, mobile, web, and console applications. Delphi's compilers use their own Object Pascal dialect of Pascal and generate native code for several platforms: Windows (x86 and x64), OS X (32-bit only), iOS (32 and 64-bit), Android, and 64 bit Linux.",
```

Comment:
This is a bit too long, I think. The suggested length is about 30 words.

From the original issue:

> The aim of this is to give someone a really quick feel for what the language is about and why it might appeal to them.

I think the first sentence accomplishes this well, whereas the second has too much detail (it's better suited to the ABOUT.md).

Maybe something like this:

> Delphi is great for developing cross-platform desktop, mobile, web, and console applications. The language is a variant of Pascal.